tests/fixtures/opensource_bitcoin_test_vector_0.json

```diff
@@ -31,9 +33,9 @@
             }
         ]
     },
-    "expected_display": "INPUTS:\n  2NAy762TReQqjNbNLdh3sK7cnje4GA4UfoW\t0.26734591 BTC\n\nOUTPUTS:\n  2NCcmVAirNBDTQYvecdDpKNJVdKJruwdUSZ\t0.12345678 BTC\n  2MyvsR5gizb6JHSECL6csBHwpW1wL8xqHp4\t0.14374378 BTC\n\nFEE: 0.00014535 BTC\n\nSIGNING AS: m/45'/1'/500'/200/26\n\nSignature Data: \n",
+    "expected_display": "INPUTS:\n  2NAy762TReQqjNbNLdh3sK7cnje4GA4UfoW\t0.26734591 BTC\tSigning as m/45'/1'/500'/200/26\n\n\nOUTPUTS:\n  2NCcmVAirNBDTQYvecdDpKNJVdKJruwdUSZ\t0.12345678 BTC\n  2MyvsR5gizb6JHSECL6csBHwpW1wL8xqHp4\t0.14374378 BTC\n\nFEE: 0.00014535 BTC\n\nSignature Data: \n",
     "expected_signature": {
-        "pubkey": "026c76fc386b6c339577eea265242eb902df43155a0eaf23af1f96c5d9f2d10f63",
+        "pubkeys": ["026c76fc386b6c339577eea265242eb902df43155a0eaf23af1f96c5d9f2d10f63","026c76fc386b6c339577eea265242eb902df43155a0eaf23af1f96c5d9f2d10f63","026c76fc386b6c339577eea265242eb902df43155a0eaf23af1f96c5d9f2d10f63"],
```

Comment:
this seems a wasteful. Since the requester has the redeemscripts, you could return the index of the public key in the script, or it's possible to brute-force check each public key in the redeem script for placement by the software that constructs the transaction